### PR TITLE
refactor(module:*): remove method parameter decorators

### PR DESCRIPTION
**Pull request by HyperLife1119**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

逐步迁移到标准装饰器，作为第一步，移除所有方法参数装饰器。

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by Nicoss54** [comment]
@HyperLife1119 review in progress :)

**Review by Nicoss54 — APPROVED** [review]
LGTM

**Comment by Nicoss54** [comment]
@HyperLife1119 please resolve the conflitcts then it willbe good

File: components/carousel/carousel.component.ts [hunk 1]
```diff
@@ -169,6 +167,8 @@ export class NzCarouselComponent implements AfterContentInit, AfterViewInit, OnD
   private pointerDelta: PointerVector | null = null;
   private isTransiting = false;
   private isDragging = false;
+  private directionality = inject(Directionality);
```
Comment (Laffery) [inline components/carousel/carousel.component.ts:170]: I wonder why use inejct here and sometimes define in the constructor

**Review by Laffery — APPROVED** [review]
LGTM

**Pull request merged**